.circleci/config.yml

```diff
@@ -38,17 +38,17 @@ jobs:
       - run:
           name: install dependencies
           command: |
-            gem install bundler:2.0.0.pre.3 && bundle _2.0.0.pre.3_ --jobs=3 --retry=3 --path vendor/bundle
```

Comment:
Oh this is interesting. I guess circle has gotten smarter about bundler? Otherwise how does it know to install it? Anyway, if this works, cool.

Reply:
Actually, I was just trying to undo the diff I needed to add with the ruby upgrade here: https://github.com/mikevallano/tmdb-moviequeue/pull/321/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L36-R41.

I changed it back to what it was before the ruby upgrade: `bundle install --jobs=3 --retry=3 --path vendor/bundle`